test/algorithms/set_operations/difference/difference.cpp

```diff
@@ -613,6 +613,13 @@ void test_all()
 
     TEST_DIFFERENCE(issue_1138, 1, 203161.751, 2, 1237551.0171, 1);
 
+    {
+        ut_settings settings;
+        settings.set_test_validity(false);
+        settings.validity_of_sym = false;
+        TEST_DIFFERENCE_WITH(issue_1226, 1, 0.238037722, 0, 0.0, 1, settings);
```

Comment:
Somehow they are reported as invalid.
While (I test in `overlay.cpp`) in the original test code, they were valid. 
I don't know the reason, it might be a false negative or a ccw issue